common-crafting.lic

```diff
@@ -88,4 +88,26 @@ module DRCC
     end
   end
   module_function :stow_crafting_item
+
+  def order_item(room, item_number)
+    DRCT.walk_to(room)
+    fput("order #{item_number}")
+    fput("order #{item_number}")
+    empty_hands
+  end
+  module_function :order_item
+
+  def empty_hands
```

Comment:
dispose can't be moved to common because it requires walk_to, which would introduce a circular dependency.